roles/openshift_manage_node/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
   register: omd_get_node
   until: omd_get_node.rc == 0
   retries: 20
-  delay: 5
+  delay: 30
```

Comment:
I think I would prefer to see the number of retries increased rather than increasing the delay here.